asserts/database.go

```diff
@@ -725,7 +725,8 @@ func CheckTimestampVsSigningKeyValidity(assert Assertion, signingKey *AccountKey
 			if !signingKey.Until().IsZero() {
 				until = fmt.Sprintf(" until %q", signingKey.Until())
 			}
-			return fmt.Errorf("%s assertion timestamp outside of signing key validity (key valid since %q%s)", assert.Type().Name, signingKey.Since(), until)
+			return fmt.Errorf("%s assertion timestamp %q outside of signing key validity (key valid since %q%s)",
```

Comment:
why %q instead of %v ?

Reply:
Used the same formatting as for the since/until timestamps.

Reply:
ah,  I missed that, sorry,  consistency is good